misago/users/signals.py

```diff
@@ -1,11 +1,22 @@
 from django.dispatch import Signal, receiver
-
+from django.contrib.auth import get_user_model
+from misago.core.utils import ANONYMOUS_IP
 
 anonymize_user_content = Signal()
 delete_user_content = Signal()
 username_changed = Signal()
-
+anonymize_old_ips = Signal()
 
 @receiver(username_changed)
 def handle_name_change(sender, **kwargs):
     sender.user_renames.update(changed_by_username=sender.username)
+
+
+UserModel = get_user_model()
+MISAGO_IP_STORE_TIME = 33300
+
+@receiver(anonymize_old_ips)
+def handle_old_ips(sender, **kwargs):
```

Comment:
Lets name receiver functions for IP's after what IP's they are anonymizing, for example `anonymize_old_registrations_ips`.